lambda/lambda.ml

```diff
@@ -727,17 +727,18 @@ let transl_extension_path loc env path =
 let transl_class_path loc env path =
   transl_path Env.find_class_address loc env path
 
-let transl_prim mod_name name =
-  let pers = Ident.create_persistent mod_name in
-  let env = Env.add_persistent_structure pers Env.empty in
-  let lid =
-    Longident.Ldot (Location.mknoloc (Longident.Lident mod_name),
-                    Location.mknoloc name)
-  in
-  match Env.find_value_by_name lid env with
-  | path, _ -> transl_value_path Loc_unknown env path
-  | exception Not_found ->
-      fatal_error ("Primitive " ^ name ^ " not found.")
+let transl_prim modname field =
+  let mod_ident = Ident.create_persistent modname in
+  let env = Env.add_persistent_structure mod_ident Env.initial in
```

Comment:
Could you use `Env.empty` instead ? I'm afraid that this version would otherwise find things it is not supposed to find like `CamlinternalMod.at_exit`

Reply:
(Maybe I'm wrong on what `Env.initial`contains, but I was afraid it would already have `Stdlib` open)

Reply:
I initially thought this as well, but in fact my understanding is that Env.initial has predef types and constructors (but not the stdlib), and Env.empty is really empty. So I think that it is more correct to use Env.initial if the CamlinternalFoo modules are allowed to mention `exn`, etc.